akka-cluster/src/multi-jvm/scala/akka/cluster/LargeMessageClusterSpec.scala

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster
+
+import scala.concurrent.duration._
+
+import akka.actor.ActorIdentity
+import akka.actor.ActorRef
+import akka.actor.Identify
+import akka.actor.PoisonPill
+import akka.cluster.ClusterEvent.UnreachableMember
+import akka.remote.testconductor.RoleName
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.testkit._
+import com.typesafe.config.ConfigFactory
+
+object LargeMessageClusterMultiJvmSpec extends MultiNodeConfig {
+  val first = role("first")
+  val second = role("second")
+  val third = role("third")
+
+  // Note that this test uses default configuration,
+  // not MultiNodeClusterSpec.clusterConfig
+  commonConfig(ConfigFactory.parseString(
+    """
+    akka {
+      loglevel = INFO
+      cluster.debug.verbose-heartbeat-logging = on
+      loggers = ["akka.testkit.TestEventListener"]
+      actor.provider = cluster
+
+      testconductor.barrier-timeout = 60 s
+
+      cluster.failure-detector.acceptable-heartbeat-pause = 3 s
+
+      remote.artery {
+        enabled = on
+
+        large-message-destinations = [ "/user/largeEcho" ]
+
+        advanced {
+          idle-cpu-level = 1
+
+          inbound-lanes = 1
+
+          maximum-frame-size = 2 MiB
```

Comment:
I tried same thing without large-message-destinations and that also works, i.e. sending them over the ordinary messages stream